src/services/environments/node-environment.service.ts

```diff
@@ -50,4 +50,9 @@ export class NodeEnvironmentService extends EnvironmentServiceBase {
     });
   }
 
+  publishProject = (name: string, path: string, _destinationPath: string): Promise<void> => {
+    return this.buildProject(name, path)
```

Comment:
The process of publishing a node project is really just copying a folder, but here you are building. This indicates to me that you should move the build step up one layer in your source hierarchy (in this case, to node-publish.command.ts), and replace this "publishProject" implementation with a call to "fileService.copyFolder".  Then your publish command will follow the pattern of "build -> publish", with each environment service method performing its specific function.